### ISSUE TITLE
Method = genderize exiting with error when lookup fails

### ISSUE THREAD

**Issue by AtisK** [ropensci/gender#28]
Hello,

Trying to execute the genderzie method on a data frame containing the name:PRISCIAN

The function is exiting with the below error: It appears that the error is thrown as the lookup failed?
Error in if (result$gender == "female") { : argument is of length zero

**Comment by lmullen** [comment]
Thanks for the bug report, @AtisK. It looks like the package does not adequately test whether it gets a result back from genderize. I'll fix this in the next release. In the meantime, you might consider wrapping the call to `gender()` with `try()`.

**Comment by AtisK** [comment]
Thank you Lincoln for your message. Yes, I will wrap it in try and run the decoding.One question though, I noticed that genderize API itself has a limit on the number of records it can process in a day. 
But the gender package using genderize doesn't have any set limits, correct? Because I combed the documentation but didn't find any reference to limits, but thought I'd confirm with you.
Regards,
Sita

Date: Fri, 24 Jul 2015 12:29:23 -0700
